LSE-509.tex

```diff
@@ -27,12 +27,11 @@
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-The Management Plan covered in this document includes the organization and management of the Systems Integration, Test and Commissioning team (here after SIT-Com) during the end stages of development, construction, and commissioning of the Rubin Observatory.
-    The SIT-Com team is composed of a core group of people working nearly full time on SIT-Com activities as well as numerous others contributing from across all of the Project subsystems and the scientific user community - both domestic and foreign.
-    This document describes the composition of this diverse team and the challeges of managing a team that are undertaking tasks spanning numerous disciplines.
-    It also lays out the management organization, leadership structure, and the team’s roles and responsibilities.
-    This document also describes the team's regular interactions, and details the workflow of how information and tasks are communicated up and down the chain of management.
-    Lastly, this document provides a high-level overview of SIT-Com scope, products and processes and acts as a structured starting point for understanding SIT-Com and provides pointers to Project system level documentation.
+The Management Plan covered in this document includes the organization and management of the Systems Integration, Test and Commissioning team (here after SIT-Com) during the end stages of development, construction, and commissioning of the Rubin Observatory.  
+The SIT-Com team is composed of a core group of people from the Rubin Construction Project working nearly full time on SIT-Com activities as well as numerous others contributing from across all of the Project subsystems and the scientific user community - both domestic and foreign.  
+This document describes the composition of the distributed SIT-Com team and the management strategy of the team who are undertaking tasks with wide ranging disciplines.  It also lays out the management organization, leadership structure, and the team members' roles and responsibilities.   
```

Comment:
D'Oh. I missed a line-break. Please add it.

Reply:
The first sentence is a bit awkward (my bad). Consider:
This document describes the composition management strategy of the distributed SIT-Com team, who are undertaking tasks with wide ranging disciplines.